cmd/internal/kube/print_snippet.go

```diff
@@ -0,0 +1,33 @@
+package kube
+
+import (
+	"github.com/postmanlabs/postman-insights-agent/cmd/internal/cmderr"
+	"github.com/postmanlabs/postman-insights-agent/printer"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
+var printHelmChartSnippetCmd = &cobra.Command{
+	Use:   "helm-sidecar-snippet",
+	Short: "Print a Helm chart container definition for adding the Postman Insights Agent to existing k8s deployment.",
+	Long:  "Print a container definition that can be inserted into a Helm Chart template to add the Postman Insights Agent as a sidecar container.",
+	RunE:  printHelmChartSnippet,
+}
```

Comment:
I think you need something like this to avoid the version header and telemetry cruft?  Probably we don't want to show that.

```
	PersistentPreRun: func(cmd *cobra.Command, args []string) {
		// This function overrides the root command preRun so we need to duplicate the domain setup.
		if rest.Domain == "" {
			rest.Domain = rest.DefaultDomain()
		}
		// Initialize the telemetry client, but do not allow any logs to be printed
		telemetry.Init(false)
	},
```

Reply:
I will remove the extra `[INFO]` log just before the snippet, but the version is getting printed as a preRun function at the root level.
Telemetry cruft is there since this was running without an Amplitude Key. The release build will have the key embedded so the log message won't be shown.

After changes it will be like this:

Case 1: Executing the command with the release build
```
[INFO] Postman Insights Agent 0.0.0
args:
- apidump
- project
...
```

Case 2: Executing the command with telemetry disabled manually
```
[INFO] Telemetry disabled via opt-out.
[INFO] Postman Insights Agent 0.0.0
args:
- apidump
- --project
...
```

Reply:
That is why for the `kube inject` command, we override the root prerun with the PersistentPreRun function I showed above, which explicitly tells the telemetry function not to output anything?

Are you saying that this doesn't work for `kube inject` either?

Reply:
Yes, I check it later, updated it in that PR along with terraform fragment command